content/_index.pt-br.md

```diff
@@ -0,0 +1,71 @@
++++
+date  = 2022-04-09
+
+title    = "Sendo bem honesto, não tem muita coisa aqui"
+subtitle = "Mas vem dar uma lida de qualquer jeito!"
+
+navTitle = "Índice"
+navIndex = 0
+
+draft = false
++++
+
+Eu sei, meu nickname é bem "l33t", e foi até gerado aleatoriamente.
+
+Não tem muita coisa para ver nesse site, mas sinta-se livre para o bisbilhotar.
+Com alguma chance, há algo de útil para você por aqui. Pretendo escrever sobre o
+que quer que venha à mente _do jeito_ que vem à mente, então se minhas palavras
+parecem delírios, divagações ou linhas de raciocínio espalhadas, provavelmente o
+são.
+
+Este site está sempre em construção. Eu não sou um desenvolvedor front-end, e
+provavelmente nunca o serei. Tudo que lhes consigo garantir são páginas sem
+quilos de JavaScript.
+
+## Código-fonte, build, hospedagem, etc.
+
+Este site está hospedado no [GitHub Pages][gh-pages], e as páginas são geradas
+pelo [Hugo][gohugo]. O código-fonte, bem como os workflows do [GitHub
```

Comment:
Tricky rules to be used here. We have:
"código-fonte" = singular
"os workflows (...)" = plural

We should check if the verb "pode" should be "podem" instead.

Reply:
I'm almost sure I can use it that way if we consider "bem como [...]" an appositive sentence, but since I couldn't find literature sustaining that case, I decided to "flatten" the sentence in [8b8f884](https://github.com/d3adb5/website/pull/29/commits/8b8f884b7c05a18a815ad59c33c4f2364d93e56b).